.github/workflows/build_nugets.yml

```diff
@@ -364,13 +364,92 @@ jobs:
+
+      # Download the previously built Nuget packages
+      - name: Clear nuget cache
+        run: dotnet nuget locals all --clear
+      - uses: actions/download-artifact@v1
+        with:
+          name: ${{matrix.package_name}}.${{steps.fetch-vw-version.outputs.VWVersion}}.nupkg
+          path: downloaded_nugets
+      - uses: actions/download-artifact@v1
+        with:
+          name: ${{matrix.package_name}}.runtime.win-x64.${{steps.fetch-vw-version.outputs.VWVersion}}.nupkg
+          path: downloaded_nugets
+      - name: List downloaded packages
+        run: ls downloaded_nugets
+
+        # Some of these commands may generate an error message because we are installing packages from local source with remote dependencies
+        # The missing dependencies are installed in later steps
+      - name: Install package
+        run: |
+          set +e
+          cd nuget/dotnet/benchmarks
+          dotnet add dotnet_benchmark.csproj package ${{matrix.package_name}} --version ${{steps.fetch-vw-version.outputs.VWVersion}} --source "${{github.workspace}}/downloaded_nugets" --no-restore
+          dotnet add dotnet_benchmark.csproj package ${{matrix.package_name}}.runtime.win-x64 --version ${{steps.fetch-vw-version.outputs.VWVersion}} --source "${{github.workspace}}/downloaded_nugets" --no-restore
+          dotnet restore dotnet_benchmark.csproj --runtime win-x64 --source "${{github.workspace}}/downloaded_nugets"
+          dotnet restore dotnet_benchmark.csproj --runtime win-x64
+          exit 0
+      - name: Build and run benchmarks
+        run: |
+          cd nuget/dotnet/benchmarks
+
+          # Build for both .NET Core and Framework
+          dotnet build dotnet_benchmark.csproj --framework net6.0 --runtime win-x64 --configuration Release --no-restore --self-contained
+          dotnet build dotnet_benchmark.csproj --framework net4.8 --runtime win-x64 --configuration Release --no-restore --self-contained
+
+          # Run with .NET Core as host process
+          # Arguments after "--" are for BenchmarkDotNet
+          dotnet run --project dotnet_benchmark.csproj --framework net6.0 --runtime win-x64 --configuration Release --no-build -- --filter '*' --runtimes net6.0 net4.8
+      - name: Print benchmark results
+        shell: bash
+        run: |
+          cd nuget/dotnet/benchmarks
+          dotnet run --project dotnet_benchmark.csproj --framework net6.0 --runtime win-x64 --configuration Release --no-build -- --info
+          echo -e '\n'
+
+          cd BenchmarkDotNet.Artifacts/results
+          for file in *-report-github.md; do
+            benchmark="${file%-report-github.md}"
+            echo "***** $benchmark *****"
+            # Remove lines not part of table (not between |...|)
+            # Remove markdown formatting **
+            # Remove first 3 columns of table
+            cat "$file" | sed '/^|.*|\s*$/!d' | sed 's/\*\*//g' | sed 's/^\(|[^|]*\)\{3\}//'
+            echo -e '\n'
```

Comment:
Could you output this to a job summary so it is easier to find? https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/

Reply:
Wow, that's a cool feature that I didn't know of!

Reply:
That's cool.